test/e2e/vpcs.e2e.ts

```diff
@@ -45,34 +45,95 @@ test('can create and delete subnet', async ({ page }) => {
   await page.goto('/projects/mock-project/vpcs/mock-vpc')
   await page.getByRole('tab', { name: 'Subnets' }).click()
   // only one row in table, the default mock-subnet
-  const rows = page.locator('tbody >> tr')
-  await expect(rows).toHaveCount(1)
-  await expect(rows.nth(0).getByText('mock-subnet')).toBeVisible()
+  const table = page.getByRole('table')
+  const rows = page.getByRole('table').getByRole('row')
+  await expect(rows).toHaveCount(2)
+
+  await expectRowVisible(table, {
+    name: 'mock-subnet',
+    'Custom Router': '—',
+    'IP Block': expect.stringContaining('10.1.1.1/24'),
+  })
 
   // open modal, fill out form, submit
-  await page.click('text=New subnet')
-  await page.fill('input[name=ipv4Block]', '10.1.1.2/24')
-  await page.fill('input[name=name]', 'mock-subnet-2')
-  await page.click('button:has-text("Create subnet")')
+  await page.getByRole('link', { name: 'New subnet' }).click()
 
-  await expect(rows).toHaveCount(2)
+  const dialog = page.getByRole('dialog', { name: 'Create subnet' })
+  await expect(dialog).toBeVisible()
 
-  await expect(rows.nth(0).getByText('mock-subnet')).toBeVisible()
-  await expect(rows.nth(0).getByText('10.1.1.1/24')).toBeVisible()
+  await dialog.getByRole('textbox', { name: 'Name' }).fill('mock-subnet-2')
+  await dialog.getByRole('textbox', { name: 'IPv4 block' }).fill('10.1.1.2/24')
 
-  await expect(rows.nth(1).getByText('mock-subnet-2')).toBeVisible()
-  await expect(rows.nth(1).getByText('10.1.1.2/24')).toBeVisible()
+  // little hack to catch a bug where we weren't handling empty input here properly
+  await dialog.getByRole('textbox', { name: 'IPv6 block' }).fill('abc')
+  await dialog.getByRole('textbox', { name: 'IPv6 block' }).clear()
```

Comment:
Confirmed this bit breaks the test without the fix for the `ipv6Block` field.

<img width="1085" alt="image" src="https://github.com/user-attachments/assets/996b00bb-2cf2-4604-b620-fc079e651273">